src/main/java/com/example/demo/ShortenUrlController.java

```diff
@@ -11,25 +13,27 @@
 public class ShortenUrlController {
 
     @Autowired
-    private UrlRepository repository;
+    private ShortenUrlService shortenUrlService;
 
     @GetMapping
-    public String helloWorld(){
-        return "Hello World from Spring Boot";
+    public ResponseEntity<String> helloWorld(){
+        String msg = "Hello World from Spring Boot";
+        HttpHeaders headers = new HttpHeaders();
+        return new ResponseEntity<String>(msg,headers, HttpStatus.OK);
     }
 
     @PostMapping("/url")
-    public Url saveUrl(@RequestBody Url url){
-
-        //TODO: Use ResponseEntity
-        return repository.save(url);
+    public ResponseEntity<Void> saveUrl(@RequestBody Url url){
+        shortenUrlService.saveUrl(url);
```

Comment:
I don't see any validation here
What if the object will be null? I don't see any exception handling as well